apis/server/volume_bridge.go

```diff
@@ -47,25 +47,23 @@ func (s *Server) createVolume(ctx context.Context, rw http.ResponseWriter, req *
 		return err
 	}
 
+	status := map[string]interface{}{}
+	for k, v := range volume.Options() {
```

Comment:
Actually there is another corner case that `size` exists in both `volume.Options()` and `volume.Size()`. 
Then what is rule to deal with this case?
Just curious, I will merge this first.